Assets/Scripts/ScoreMechanism.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class ScoreMechanism : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        //DontDestroyOnLoad(gameObject);
+        StartCoroutine(Test());
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        int score = PlayerPrefs.GetInt("score");
+        Debug.Log(score);
```

Comment:
How to see this printed out on console?

Reply:
This script is not working as of now. I'm directly displaying the score on existing scene. This file was created to display the score on a new scene. Given the time constraint, I'm pushing it post midterm and will continue working on this file

Reply:
I see. I'd accept the PR as of now. Later in the meantime, we'd better include files that are relevant to the commit and only those that actually bring changes to the project. Thanks for the input!